src/types.ts

```diff
@@ -21,4 +21,11 @@ export interface SysProps {
 
 export interface CollectionItem {
   sys: SysProps;
+  __typename?: string;
+}
+
+export enum MessageAction {
```

Comment:
FYI:
if you define a enum as `const enum` it would be transformed during the build process and the outcome would be smaller & treeshakeable.
Con: you need to iterate over it in a different way

Reply:
Interesting! 🤯